sdks/java/core/src/test/java/org/apache/beam/sdk/coders/BigIntegerCoderTest.java

```diff
@@ -96,4 +96,19 @@ public void testStructuralValueReturnTheSameValue() {
     Object actual = TEST_CODER.structuralValue(expected);
     assertEquals(expected, actual);
   }
+
+  @Test
+  public void testStructuralValueDecodeEncodeEqual() throws Exception {
+    for (BigInteger value : TEST_VALUES) {
+      CoderProperties.structuralValueDecodeEncodeEqual(TEST_CODER, value);
+    }
+  }
+
+  @Test
+  public void testStructuralValueConsistentWithEquals() throws Exception {
```

Comment:
For `structuralValueConsistentWithEquals` it is also good to check non-equal members. So the test at `BitSetCoderTest.testStructuralValueConsistentWithEquals` is a good example of how to use it.

Reply:
Ok, I'll change this.

Reply:
Updated